train.py

```diff
@@ -40,6 +40,12 @@
         default=10000,
         type=int,
     )
+    parser.add_argument(
+        "--logging",
```

Comment:
maybe `optimization-log-path` (or something similar) to be more explicit and avoid confusion vs normal training?